src/FilePathUtility.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex;
+
+class FilePathUtility
```

Comment:
A suggestion: rename `FilePathUtility` to `Path`. Then rename `relativizePath` to `relativize` and `concatenatePathParts` to `concatenate`. I think the code would be easier to read: `$path->relativize(...)`. WDYT?

Reply:
Sounds good.